build.yaml

```diff
@@ -300,6 +300,7 @@ filegroups:
   - include/grpc/status.h
   - include/grpc/support/workaround_list.h
   headers:
+  - src/core/gpr/vector.h
```

Comment:
Should this move to GPR?  It's a C++ thing, so I would think it would live in lib/support.